server/controllers/customer.js

```diff
@@ -21,6 +23,7 @@ export default {
     )
 
     const savedCustomer = await customer.save()
+    update.updatingVolunteer(req)
```

Comment:
why not pass the `customer._id` and `customer.fields` specifically instead of the whole `req` object?